packages/web-react/src/components/FileUploader/useFileUploaderStyleProps.ts

```diff
@@ -0,0 +1,83 @@
+import classNames from 'classnames';
+import { useClassNamePrefix } from '../../hooks';
+import { Validation } from '../../types';
+
+export interface FileUploaderStyleProps extends Validation {
+  isDragAndDropSupported?: boolean;
+  isDisabled?: boolean;
+  isDragging?: boolean;
+}
+
+export interface FileUploaderStyleReturn {
+  /** className props */
+  classProps: {
+    root: string;
+    input: {
+      root: string;
+      label: string;
+      input: string;
+      dropLabel: string;
+      helper: string;
+      link: string;
+      message: string;
+      dropZone: {
+        root: string;
+        label: string;
+      };
+    };
+    list: string;
+    attachment: {
+      root: string;
+      button: string;
+    };
+  };
+}
+
+export const useFileUploaderStyleProps = (props?: FileUploaderStyleProps): FileUploaderStyleReturn => {
+  const fileUploaderClass = useClassNamePrefix('FileUploader');
+  const fileUploaderHasDragAndDropClass = 'has-drag-and-drop';
+  const fileUploaderInputClass = `${fileUploaderClass}Input`;
+  const fileUploaderInputValidationClass = `${fileUploaderInputClass}--${props?.validationState}`;
+  const fileUploaderInputDraggingClass = 'is-dragging';
+  const fileUploaderInputLabelClass = `${fileUploaderInputClass}__label`;
+  const fileUploaderInputLabelRequiredClass = `${fileUploaderInputClass}__label--required`;
+  const fileUploaderInputInputClass = `${fileUploaderInputClass}__input`;
+  const fileUploaderInputDropZoneClass = `${fileUploaderInputClass}__dropZone`;
+  const fileUploaderInputDropZoneLabelClass = `${fileUploaderInputDropZoneClass}Label`;
+  const fileUploaderInputDropLabelClass = `${fileUploaderInputClass}__dragAndDropLabel`;
+  const fileUploaderInputHelperClass = `${fileUploaderInputClass}__helperText`;
+  const fileUploaderInputLinkClass = `${fileUploaderInputClass}__link`;
+  const fileUploaderInputLinkUtilityClasses = ['link-primary', 'link-underlined'];
+  const fileUploaderInputMessageClass = `${fileUploaderInputClass}__message`;
```

Comment:
This has changed to `__validationText`.


```suggestion
  const fileUploaderInputMessageClass = `${fileUploaderInputClass}__validationText`;
```

Reply:
fixed